lib/components/user/mobility-profile/mobility-wizard.tsx

```diff
@@ -0,0 +1,30 @@
+import { useIntl } from 'react-intl'
+import React from 'react'
+
+import Wizard, { WizardProps } from '../wizard'
+
+/**
+ * This component is the mobility profile wizard.
+ */
+const MobilityWizard = ({
+  activePaneId,
+  formikProps
+}: WizardProps): JSX.Element => {
+  const intl = useIntl()
+
+  const title = intl.formatMessage({
+    id: 'components.MobilityProfile.title'
+  })
+
+  return (
+    <Wizard
+      activePaneId={activePaneId}
+      formikProps={formikProps}
+      pages={['mobility1', 'mobility2']}
```

Comment:
Can we change these to "assistive devices" and "limitations"?

Reply:
Yes, I did in https://github.com/opentripplanner/otp-react-redux/pull/1090/commits/b1ad318057f5692261dcef584e2f0bc224f0abb5.